src/tooltip/index.es

```diff
@@ -1,21 +1,54 @@
-import React from 'react';
+import React, { useCallback, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
+import { usePopper } from 'react-popper';
 
 import { arrayToHash } from 'nebenan-helpers/lib/data';
+import { getPopperOptions } from '../feature_alert/utils';
 
 const knownTypes = arrayToHash(['left', 'right', 'top', 'bottom']);
 
 
 const Tooltip = (props) => {
+  const [isOpen, setOpen] = useState(false);
   const { type, text, children, ...cleanProps } = props;
   const selectedType = knownTypes[type] ? type : 'top';
-  const className = clsx(`c-tooltip c-tooltip-${selectedType}`, props.className);
+  const className = clsx('c-tooltip', props.className, {
+    'is-active': isOpen,
+  });
+
+  const handleOpen = useCallback((event) => {
+    if (event) event.stopPropagation();
+    if (isOpen) return;
+    setOpen(true);
+  }, [isOpen]);
+
+  const handleClose = useCallback(() => {
+    if (!isOpen) return;
+    setOpen(false);
```

Comment:
do we really need the `isOpen` check? Not sure if setting the same value again would trigger a render

`setOpen(false)` -> render
`setOpen(false)` -> does not render?